### ISSUE TITLE
import  for  GraphQLField is missing

### ISSUE THREAD

**Issue by mheidt** [graphql-java-generator/graphql-maven-plugin-project#135]
Hi Etienne,

I tried to upgrade from 1.18.1 to latest but from 1.18.3 to 1.18.4 the server code generation failed.

In all classes that have this:
public static enum Field implements GraphQLField,

the 
import com.graphql_java_generator.GraphQLField;
is missing.

The GraphQLField class can be found in the target folder.

Those are my settings (generateServerCode):
<addRelayConnections>false</addRelayConnections>
<copyRuntimeSources>true</copyRuntimeSources>
<generateBatchLoaderEnvironment>true</generateBatchLoaderEnvironment>
<packageName>de.lsy.cmon.aggregator.graphql</packageName>
<separateUtilityClasses>true</separateUtilityClasses>
<skipGenerationIfSchemaHasNotChanged>true</skipGenerationIfSchemaHasNotChanged>

**Comment by etienne-sf** [comment]
Ok, I'll check that. 

I had to change the import management, to correct an issue. 
But there are two many combinations of parameters to test them all. 

I started the 2.x branch, for compatibility with spring-graphql. In this branch, less combinations will be accepted (some parameters will be removed). 
This will allow extensive tests. 

Étienne

**Comment by etienne-sf** [comment]
Hello,

I can't find where to find the ´public static enum Field implements GraphQLField´ you described. 

Isn't it coming from your GraphQL schéma ?
Or from a custom templates ?

If no, at least, is it in client or server mode ?

Étienne

**Comment by etienne-sf** [comment]
BTW, did you try with the last version, 1.18.6 ?

**Comment by mheidt** [comment]
yes..and then I went step by step from 18.1 to 2, 3 and 4 broke

**Comment by etienne-sf** [comment]
GraphQLField has been removed in the 1.18.6 release. 
And it is nowhere in the project code. 

So, if you encounter this error with the 1.18.6 release, my opinion is that is comes from some code of yours. 

Or perhaps you didn't do a ´mvn clean´ or ´gradlew clean´ after switching to the 1.18.6 release ?

**Comment by mheidt** [comment]
Yes, you removed the $imports and my java.vm for the batchloader was still using
${batchLoader.type.classSimpleName}
